src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray, Controller } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { InputField } from "components/atoms/InputField";
+import { MAX_QUESTIONS_NUMBER } from "settings";
+import "./PollBox.scss";
+
+export type Question = {
+  name: string;
+};
+
+export type PollValues = {
+  topic: string;
+  questions: Question[];
+};
+
+export interface PollBoxProps {
+  onSubmit: (args: PollValues) => void;
+}
+
+const defaultQuestion: Question = { name: "" };
+const defaultValues = {
+  topic: "",
+  questions: [defaultQuestion, defaultQuestion],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ onSubmit }) => {
+  const { control, handleSubmit, reset, watch } = useForm<PollValues>({
+    defaultValues,
+  });
+  const { fields, append } = useFieldArray({ name: "questions", control });
+
+  const onCustomSubmit = handleSubmit((data) => {
+    onSubmit(data);
+    reset();
+  });
+
+  const isDisabled = useMemo(() => {
+    const [question1, question2] = watch("questions");
+    return !(watch("topic") && question1.name && question2.name);
+  }, [watch]);
+
+  const addChoice = useCallback(() => append(defaultQuestion), [append]);
+  const showAppend = useCallback(
+    (index) =>
+      index + 1 === fields.length && MAX_QUESTIONS_NUMBER > fields.length,
+    [fields]
+  );
```

Comment:
to make sure not render "plus" button on the 8th choice. 
Feel free to tip better solution, I suppose it should be

Reply:
So, what's a difference, if we replace it with `const showAppend = MAX_QUESTIONS_NUMBER > fields.length`?

Reply:
Well, I will be rendering "plus" button for EVERY question instead of the last one.
There is an ability to add a new item to Question[] from 2nd to 7th and "plus" should be in the right of the last question. Propose the better view to handle this case